apps/azure-functions-api/api/internal.yaml.template

```diff
@@ -0,0 +1,82 @@
+openapi: 3.0.1
```

Comment:
what's the reason behind this file?

Reply:
why fo we need a template to do this?

Reply:
Aside `ProblemJson`, which other shared types do you think we need atm? 

This would be feasible once we don't have to import the whole `io-functions-commons` package (which has third party javascript dependencies) just to referer OpenAPI shared types.

"You wanted a banana but what you got was a gorilla holding the banana and the entire jungle"

Reply:
Remove templating in 3018cc6